sdks/python/apache_beam/dataframe/io.py

```diff
@@ -59,6 +59,11 @@ def read_fwf(path, *args, **kwargs):
 
 
 def read_json(path, *args, **kwargs):
+  if 'nrows' in kwargs:
+    raise NotImplementedError('nrows not yet supported')
+  elif kwargs.get('lines', False):
+    # Work around https://github.com/pandas-dev/pandas/issues/34548.
+    kwargs = dict(kwargs, nrows=1 << 63)
```

Comment:
The motivation isn't clear to me from that issue, could you clarify?

Reply:
Due to a bug in the code, the entire file is read into memory all at once unless both chunksize *and* nrows are set.

Reply:
Ah ok, thanks